CHANGELOG.md

```diff
@@ -3,16 +3,84 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Changed
+[#446](https://github.com/plotly/dash-table/pull/446)
+- Table API rework
+#### NEW
+    - `column.sort_as_none`: Allows sorting behavior customization.
+        Accepts an array of string, number or booleans.
+    - `sort_as_none`: Allows sorting behavior customization.
+        Accepts an array of string, number or booleans.
```

Comment:
as a replacement for `sorting_treat_empty_string_as_none` - or put that in **REMOVED**